lib/tasks/sync.rake

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+# frozen_string_literal = true
+namespace :pulfalight do
+  desc "Sync all pulfalight resources"
+  task sync_resources: :environment do
+    @logger = Logger.new(STDOUT)
+    @logger.info "Finding resources with archival collection codes"
+    query_service = ChangeSetPersister.default.query_service
+    resources = query_service.custom_queries.find_by_property(property: :archival_collection_code, value: [], model: ScannedResource, lazy: true)
+    @logger.info "Resource search complete"
+    resources.each do |resource|
```

Comment:
There are lots of resources that have the same archival collection code, I think this would create several archival collection jobs with the same arguments. Thoughts on how to prevent that?